### PR TITLE
Update compiler activation for 1.17.6

### PR DESCRIPTION
**Pull request by mariusvniekerk**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
@conda smithy rerender

**Comment by mariusvniekerk** [comment]
@conda-forge-admin please rerender

**Comment by mariusvniekerk** [comment]
@conda-forge-admin please rerender

File: conda-forge.yml [hunk 1]
```diff
@@ -3,3 +3,5 @@ build_platform:
   linux_aarch64: linux_64
   linux_ppc64le: linux_64
   osx_arm64: osx_64
+test: native
+build_with_mambabuild: false
```
[the remark this answers is not in the document]
Comment (xhochy) [reply conda-forge.yml:7]: Ah, I see, the `osx` constraint.

**Pull request merged**